plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -620,7 +620,7 @@ protected JdbcOutputTableHandle createTable(ConnectorSession session, ConnectorT
 
             RemoteTableName remoteTableName = new RemoteTableName(Optional.ofNullable(catalog), Optional.ofNullable(remoteSchema), remoteTargetTableName);
             String sql = createTableSql(remoteTableName, columnList.build(), tableMetadata);
```

Comment:
Maybe `createTableSql` should return a list of sqls?

Reply:
Yes, can, I was thought of this at the beginning, but I didn't want to modify `createTableSql` here,  so I added an extra method to get "extra" command while creating table.
Actually we can make it without changing the `createTableSql`.  We could build the create table statements using the result of `createTableSql`, and expose a list of sqls, just as `getExtraTableAndColumnComments` does.
Adding a method here also could let supporting comment on column easier, we do not modify the response of `getColumnDefinitionSql` after this. 
I will modify the code and please help to check if it is doable, thanks!

Reply:
Looked at the code again, if the `execute for the sqls` no need to  add transaction, seems just need to modify added `execute` method, would like to hear more advices before going to next step.:)

Reply:
I wanted to use `createTableSql` because there are different remote databases and some of them may support setting the comment in `CREATE TABLE` statement like we do in Trino. Notice that we try to follow the ANSI SQL so there is some likelihood that such database exists.

Reply:
For the databases they support setting the comment in `CREATE TABLE` just leave the `getExtraTableAndColumnComments` return empty list, for others that support syntax like `comment on aaa is xxx` can return the list of comments in the added function.